### PR TITLE
SHOP-14444: implemented flex-grid

### PR DESCRIPTION
**Pull request by Yichang812**

### CODE REVIEW

File: _variables.scss [hunk 1]
```diff
@@ -61,3 +61,49 @@ $lineThick:       2px;
...
+$containerMaxWidth: 1344px;
+
+$breakPointsLg: 1024px;
+
+$gridSmColumns: 8;
+$gridLgColumns: 12;
+
+// New color names
+
+$white:       #fff;
```
Comment (winminsoe) [inline _variables.scss]: why not use this? https://github.com/zalora/global-scss/blob/ee50c86b7aa94c14380f428d44e7a139f8fb3127/_variables.scss#L6

when doing the style guide, we wanted to follow the design pattern. Every Website have primary color and secondary color and so on. And the variable name should not reflect the actual colour.

File: _variables.scss [hunk 2]
```diff
@@ -61,3 +61,49 @@ $lineThick:       2px;
...
+$lightGrey:  #F4F4F4;
+$mediumGrey: #dfdfdf;
+$darkGrey:   #6d6e6e;
+$black:       #000;
+
+$error:       #ba4242;
```
Comment (winminsoe) [inline _variables.scss]: same for here. we should use the same error Color. No?. https://github.com/zalora/global-scss/blob/ee50c86b7aa94c14380f428d44e7a139f8fb3127/_variables.scss#L15

File: desktop/components/_forms.scss [hunk 3]
```diff
@@ -44,6 +44,15 @@ textarea {
       background: url('../../../images/local/checkout/red_error_icon-02112015.png') no-repeat right 10px center $colorLightPrimary;
     }
 
+    &.success {
+    border: 1px solid #00c8b4;
+    background: url("../../../images/local/icons/svg/icon_check-light.svg") no-repeat right 10px center #fff;
+    }
+
+    &.progressing {
```
Comment (penmanglewood) [inline desktop/components/_forms.scss]: How could this be empty?

File: desktop/components/_forms.scss [hunk 4]
```diff
@@ -44,6 +44,15 @@ textarea {
       background: url('../../../images/local/checkout/red_error_icon-02112015.png') no-repeat right 10px center $colorLightPrimary;
     }
 
+    &.success {
+    border: 1px solid #00c8b4;
```
Comment (penmanglewood) [inline desktop/components/_forms.scss]: Indentation

**Review by penmanglewood — CHANGES_REQUESTED** [review]

**Review by penmanglewood — APPROVED** [review]

**Pull request merged**